manifests/init.pp

```diff
@@ -36,6 +36,8 @@
 # $enable_ostree::        Boolean to enable ostree plugin. This requires existence of an ostree install.
 #                         type:boolean
 #
+# $github_username::      Github username to add remotes for
+#
```

Comment:
Could we enable arbitrary pluigns? Like a param here that takes an array of plugins to enable? Then you can do:

`katello_devel::plugin { $enable_plugins:}` and it'll create them all.  Not sure how this would work with the inconsistencies with foreman plugins, foreman_docker gem is in a foreman-docker repo.